src/templates/views/my-trips.njk

```diff
@@ -19,9 +37,10 @@
     <img src="/static/icons/createtrip.svg">
   </a>
   {% endif %}
+  
   {% for trip in trips %}
-  <a href="/{% if trip.leader === 1 %}leader/{% endif %}trip/{{ trip.id }}" class=trip-card>
-    <img class=club-logo src="{{ trip.iconPath }}">
+  <a href="/{% if trip.leader === 1 %}leader/{% endif %}trip/{{ trip.id }}" class="trip-card">
+    <img class="club-logo" src="{{ trip.iconPath }}">
```

Comment:
No need for this change, it's [perfectly legal](http://lofi.limo/blog/write-html-right) to not wrap attributes in quotes